src/components/UiKit/FormInputs/PasswordInput/index.jsx

```diff
@@ -5,7 +5,7 @@ import EyeHidden from "@components/UiKit/Icons/EyeHidden/index.jsx"
 import styles from "./password.module.scss"
 
 const PasswordInput = forwardRef(({ isValid = true, errorMessage, ...rest }, ref) => {
-  const [isInputFocused, setIsInputFocused] = useState(false)
+  const [isInputFocused, setIsInputFocused] = useState(rest?.defaultValue)
```

Comment:
!!! don't use rest?.defaultValue inside useState
better to extract necessary props explicitly rather than accessing them through rest?.defaultValue
problems u will have with rest params:
- Unexpected behavior
- Less readable code
- Potential refactoring issues